README.md

```diff
@@ -8,9 +8,13 @@ Included in this repo is a set of bots that disseminate information to Beanstalk
 - **Contract Bot** - sends a message detailing each interaction with the Beanstalk contact
 
 ### Running locally
+First, install the necessary requirements using `pip3 install -r beanstalk-py/requirements.txt`.
+
 To run the bots locally you will need to set several environment variables with your own keys.
 Environment variables necessary:
-- `ETH_CHAIN_API_KEY`
-- `SUBGRAPH_API_KEY`
+- `ALCHEMY_ETH_API_KEY`
 - `DISCORD_BOT_TOKEN` (`DISCORD_BOT_TOKEN_PROD` for prod application)
-- `TELE_BOT_KEY` (`TELEGRAM_BOT_TOKEN_PROD` for prod application)
+- `DISCORD_BASIN_BOT_TOKEN` (`DISCORD_BASIN_BOT_TOKEN_PROD` for prod application)
+- (`TELEGRAM_BOT_TOKEN_PROD` for prod application)
+
+Create an `env.dev` file using the provided example and place your varaibles there. Then, execute `./dev.sh <module>`. For example, to run the main set of bots, execute `./dev.sh bots.discord_bot`.
```

Comment:
what is the reason for changing the handling of tokens for these three tokens and have they been set up to function on the server prod instances using systemd?

Reply:
The only thing I changed was with respect to local development. Instead of having to export the required variables globally or in the session that starts the bots, it gets taken care of in that wrapper script I created, which pulls those variables from another file